src/main/java/storage/AuthorListDeserializer.java

```diff
@@ -20,6 +21,7 @@ class AuthorListDeserializer implements JsonDeserializer<AuthorList> {
     @Override
     public AuthorList deserialize(JsonElement json, Type typeOfAuthorList, JsonDeserializationContext context)
             throws JsonParseException {
+        Ui ui = new Ui();
```

Comment:
Would it be better if these `Ui` methods were made `static` instead? This way there is no need to instantiate a new `Ui` object every time we want to print something.